tests/backup/backup_helper.go

```diff
@@ -2026,3 +2028,28 @@ func MapToKeyValueString(m map[string]string) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// LicenseCountCheck verifies the license count for backup
+func LicenseCountCheck(ctx context.Context, licenseInspectRequestObject *api.LicenseInspectRequest, expectedLicenseCount int64) error {
+	licenseCountCheck := func() (interface{}, bool, error) {
+		licenseInspectResponse, err := Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+		if err != nil {
+			return "", false, err
+		}
+		licenseResponseInfoFeatureInfo := licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+		if licenseResponseInfoFeatureInfo[0].Consumed == expectedLicenseCount {
+			return "", false, nil
+		}
+		return "", true, nil
+	}
+	_, err := task.DoRetryWithTimeout(licenseCountCheck, licenseCountUpdateTimeout, licenseCountUpdateRetryTime)
+	if err != nil {
+		licenseInspectResponse, err1 := Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+		if err1 != nil {
+			return err1
```

Comment:
Here, you can combine `err` and `err1` and return it, as both errors may be meaningful in debugging:
```go
return fmt.Errorf("licenseCountCheck failed with error: %v. Debug attempt 'Inst().Backup.InspectLicense' failed with error: %v", err, err1)
```

Reply:
done